DTS-STN/Alpha-Site
# Added required field if yes is selected.

# Description

[Required disability](https://trello.com/c/Ekn7HPRY/328-for-the-disability-question-on-the-sign-up-page-change-logic-and-text-for-the-follow-up-question-asking-about-what-accommodation)

If the user selects "yes" for the disability question in the signup page, then the text field is required.

## Acceptance Criteria

Must enter some text in the yes text box.

## Test Instructions

1. Navigate to branch name: `required-disability`
2. Go to the sign up page
3. Submit the form as is. (all empty)
4. Should see 4 errors.
5. Now, select yes for the question: "Do you identify as a person with a disability?"
6. Re-submit the form.
7. Should see 5 errors.

## Help Requested

- any constructed feedback is greatly appreciated!

## Checklist
